src/CommunityToolkit.Maui/Alerts/Toast/Toast.macios.cs

```diff
@@ -0,0 +1,66 @@
+﻿using CommunityToolkit.Maui.Core.Views;
+using CoreGraphics;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Alerts;
+
+public partial class Toast
+{
+	static ToastView? nativeToast;
+	static readonly SemaphoreSlim semaphoreSlim = new(1, 1);
+
+	/// <summary>
+	/// Dismiss Toast
+	/// </summary>
+	public virtual async partial Task Dismiss(CancellationToken token)
+	{
+		if (nativeToast is null)
+		{
+			return;
+		}
+
+		await semaphoreSlim.WaitAsync(token);
+
+		try
+		{
+			token.ThrowIfCancellationRequested();
+
+			nativeToast.Dismiss();
+		}
+		finally
+		{
+			semaphoreSlim.Release();
+		}
+	}
+
+	/// <summary>
+	/// Show Toast
+	/// </summary>
+	public virtual async partial Task Show(CancellationToken token)
+	{
+		await Dismiss(token);
+		token.ThrowIfCancellationRequested();
+
+		var cornerRadius = CreateCornerRadius();
+		var padding = GetMaximum(cornerRadius.X, cornerRadius.Y, cornerRadius.Width, cornerRadius.Height) + ToastView.DefaultPadding;
+		nativeToast = new ToastView(Text,
+											UIColor.LightGray,
+											cornerRadius,
+											UIColor.DarkTextColor,
+											UIFont.SystemFontOfSize(UIFont.LabelFontSize),
+											1,
```

Comment:
We need to set the character spacing to `0.0d` to use the default font spacing 
```suggestion
											0.0d,
```

![](https://user-images.githubusercontent.com/13558917/151896223-79e82403-6feb-4f36-9973-b971665dc97d.png)

Reply:
I will run my mac VM tomorrow and check both snackbar and toast. I hope we'll merge it tomorrow

Reply:
sorry for a delay. I made some changes to be more consistent in style (background, font size, font color). I also found a way to customize toast (at least font size), so I added a new property for it.
I verified Apple version. it looks good. I will verify Android and WinUI.